pants.ini

```diff
@@ -147,7 +147,7 @@ structs_exports: {
 
 [gen.thrift-java]
 gen_options_map: {'hashcode': ''}
```

Comment:
Every shard with a non-mysterious error output fails because of this; ie:
```
...
Error: unknown option java:hashcode
...
```
Is due to this - kill it.

I went through this almost a year ago over in apache here: https://reviews.apache.org/r/64290/diff/4?page=2#20

Reply:
That did it! Thank you John!! 🎉